CRUD-APP/user/AddUser.js

```diff
@@ -20,8 +20,13 @@ const AddUser = () => {
   const onSubmit = async e => {
     e.preventDefault();
     await axios.post("http://localhost:3003/users", user);
-    history.push("/");
-  };
+    .then(response =>{
+      history.push("/");
+    })
+    .catch(err => {
+    console.log(err);
+    });
+    };

```

Comment:
This is not how you use **.then().catch()**. This is a promise and you are using async-await for api calls.
Either use promise or use async-await for API calls.

In promise you will get catch block in chain. So no need to use try-catch with promise. 
But with async-await you need to wrap it with try-catch block.